src/content.js

```diff
@@ -1,433 +1,382 @@
+}
+
+/**
+ * Clean the element from character, tags, etc... added by the plugin logic.
+ *
+ * @method cleanInternals
+ * @param  {HTMLElement} element The element to process.
+ */
+export function cleanInternals (element) {
+  // Uses extract content for simplicity. A custom method
+  // that does not clone the element could be faster if needed.
+  element.innerHTML = extractContent(element, true)
+}
+
+/**
+ * Extracts the content from a host element.
+ * Does not touch or change the host. Just returns
+ * the content and removes elements marked for removal by editable.
+ *
+ * @param {DOM node or document framgent} Element where to clean out the innerHTML. If you pass a document fragment it will be empty after this call.
+ * @param {Boolean} Flag whether to keep ui elements like spellchecking highlights.
+ * @returns {String} The cleaned innerHTML of the passed element or document fragment.
+ */
+export function extractContent (element, keepUiElements) {
+  const innerHtml = (element.nodeType === nodeType.documentFragmentNode
+    ? getInnerHtmlOfFragment(element)
+    : element.innerHTML
+  )
+  .replace(zeroWidthNonBreakingSpace, '') // Used for forcing inline elments to have a height
+  .replace(zeroWidthSpace, '<br>') // Used for cross-browser newlines
+
+  const clone = document.createElement('div')
+  clone.innerHTML = innerHtml
+  unwrapInternalNodes(clone, keepUiElements)
+
+  return clone.innerHTML
+}
+
+export function getInnerHtmlOfFragment (documentFragment) {
+  const div = document.createElement('div')
+  div.appendChild(documentFragment)
+  return div.innerHTML
+}
+
+/**
+ * Create a document fragment from an html string
+ * @param {String} e.g. 'some html <span>text</span>.'
+ */
+export function createFragmentFromString (htmlString) {
+  const fragment = document.createDocumentFragment()
+
+  $('<div>').html(htmlString).contents().each((i, el) => {
+    fragment.appendChild(el)
+  })
+
+  return fragment
+}
+
+export function adoptElement (node, doc) {
+  return node.ownerDocument !== doc
+    ? doc.adoptNode(node)
+    : node
+}
+
+/**
+ * This is a slight variation of the cloneContents method of a rangyRange.
+ * It will return a fragment with the cloned contents of the range
+ * without the commonAncestorElement.
+ *
+ * @param {rangyRange}
+ * @return {DocumentFragment}
+ */
+export function cloneRangeContents (range) {
+  const rangeFragment = range.cloneContents()
+  const parent = rangeFragment.childNodes[0]
+  const fragment = document.createDocumentFragment()
+  while (parent.childNodes.length) fragment.appendChild(parent.childNodes[0])
+  return fragment
+}
+
+/**
+ * Remove elements that were inserted for internal or user interface purposes
+ *
+ * @param {DOM node}
+ * @param {Boolean} whether to keep ui elements like spellchecking highlights
+ * Currently:
+ * - Saved ranges
+ */
+export function unwrapInternalNodes (sibling, keepUiElements) {
+  while (sibling) {
+    const nextSibling = sibling.nextSibling
```

Comment:
This is a bit of a stylistic question whether to do this or not, but it's something I've carried over from functional programming.
For classic constants I'd still use the uppercase convention e.g. `const REFRESH_INTERVALL = 100`, but it doesn't hurt to use const for regular variables as well.
Code that doesn't modify variables generally tends to have less side effects, to run faster and to be more readable. In this case `const` is simply used as the default variable type so redeclarations can never happen accidentally or unintentionally. If you need to modify one at a later point you're always free to change things to `let` or `var`, but then your making a conscious decision, otherwise the compiler will warn you.
It's meant to nudge everyone towards best practices.